cpp/include/cugraph_c/graph.h

```diff
@@ -165,13 +256,66 @@ cugraph_error_code_t cugraph_mg_graph_create(
   cugraph_graph_t** graph,
   cugraph_error_t** error);
 
+/**
+ * @brief     Construct an MG graph
+ *
+ * @param [in]  handle          Handle for accessing resources
+ * @param [in]  properties      Properties of the constructed graph
+ * @param [in]  vertices        List of device arrays containing the unique vertex ids.
+ *                              If NULL we will construct this internally using the unique
+ *                              entries specified in src and dst
+ *                              All entries in this list will be concatenated on this GPU
+ *                              into a single array.
```

Comment:
What happens if vertices are explicitly provided but for this GPU, there is no vertex ID? This caused a bug in our C++ API (this is no longer a problem as we use std::optional). Can we handle this in the C API?